### PR TITLE
machine/ociartifact: Include image name in error

### PR DESCRIPTION
**Pull request by cgwalters**
If we hit an error here, it will be really useful to know

- That we're trying to fetch a container image
- Which image we're trying to fetch

```release-note
None
```

### CODE REVIEW

**Comment by cgwalters** [comment]
With this I now get

```
Error: failed to pull quay.io/baude/stage-podman-machine@sha256:e9fc482fc0e1903329a98d41030fbaada99e7418b6c7cbf1fc6642fc797a7fe4: no DefaultPolicyJSONPath defined and no local overwrites found: ["stat /Users/walters/.config/containers/policy.json: no such file or directory" "stat /etc/containers/policy.json: no such file or directory"]
```

which is definitely way longer, but also again at least helps me to understand this has something to do with a container image fetch.

**Comment by baude** [comment]
LGTM, you have other hurdles to clear regarding release note, etc

**Comment by mheon** [comment]
Code LGTM

**Comment by rhatdan** [comment]
Need a test of [no new tests needed]

**Comment by TomSweeneyRedHat** [comment]
Code LGTM
as Dan noted, you need a test or you need to add the label "[NO NEW TESTS NEEDED" to your commit message. 
I've fixed the release note label for you.

**Comment by cgwalters** [comment]
Thanks all...maybe some day I will contribute enough to podman to remember these hurdles.  I get why they exist, but definitely a bit of a downer for random drive-by changes.  Maybe at least the tests-needed thing could change to be a label (that anyone could trivially add without force-pushing to the branch)?

**Comment by mheon** [comment]
Making it a label would also require explicit addition by maintainers, instead of allowing contributors to do it, which I kind of like - it means that someone with merge authority had to decide that a PR was OK without tests.

**Comment by mheon** [comment]
@edsantiago PTAL

**Comment by edsantiago** [comment]
PTAL at what? Suggesting a test, or the label thing? Assuming it's the label: I'm tentatively OK with that. Looking back at the script history, I see no discussion of that possibility, so I have no idea why I didn't do that in the first place. I'll look into it.

**Comment by mheon** [comment]
Label thing. Sorry if not clear. Long day.

**Review by vrothberg — COMMENTED** [review]
/lgtm
/hold

**Review by Luap99 — APPROVED** [review]
/hold cancel

**Comment by rhatdan** [comment]
It would be nice if it was a label thing, to retrigger a failed test when the label is set.

Right now I go in an add the [NO NEW TESTS NEEDED] flag, but still have to tell the users to repush. No way for us to retrigger the tests.

**Pull request merged**
